samples/server/petstore/kotlin-springboot-delegate/src/main/kotlin/org/openapitools/api/PetApi.kt

```diff
@@ -59,7 +59,7 @@ interface PetApi {
             consumes = ["application/json", "application/xml"]
     )
     fun addPet(@Parameter(description = "Pet object that needs to be added to the store", required = true) @Valid @RequestBody pet: Pet,serverHttpRequest: ServerHttpRequest): ResponseEntity<Pet> {
-        return getDelegate().addPet(pet, serverHttpRequest)
+        return getDelegate().addPet(pet, )
```

Comment:
hmm... why there are changes to kotlin samples?

in the latest msater, the samples are up-to-date. can you please merge the latest master into your branch and regenerate the samples?

Reply:
Hmm, that's strange, I already did that and merging with master is what caused the kotlin examples to change for me. Perhaps it's an environment thing (I'm using the devcontainers environment and `bin/generate-samples.sh`) , or i'll try rollback to an earlier commit where the changes don't happen.